deploy-as-code/helm/environments/works-dev.yaml

```diff
@@ -182,6 +182,13 @@ estimate-service:
 loi-service:
   tracing-enabled: true
 
+works-inbox-service:
+  tracing-enabled: true
+  es-search-pagination-default-limit: 50

```

Comment:
If we are not using ES, then remove it from the config.

Reply:
will discuss on this...